inhouse_bot/matchmaking_logic/evaluate_game.py

```diff
@@ -51,46 +47,24 @@ async def get_team_mmr(team: List[GameParticipant]) -> int:
             value = mmrValues[rankString]
             mmr += value
         else:
-            # We assume players without function at a silver level (TODO: Come up with some reasoning on what value this should be)
-            mmr += 1400
-    print("\n")
+            # We assume players without rank play at a silver level
+            # (TODO: Silver 4 can still be too high for unranked players, consider going to bronze/iron if matchmaking is really bad)
+            mmr += mmrValues["SILVERIV"]
     return mmr
 
 
-async def evaluate_game(game: Game) -> int:
+async def evaluate_game(game: Game) -> Dict[str, int]:
```

Comment:
I prefer something like a dataclass here for "GameInfo" instead of a dictionary[str, int] because the type is stronger.